docs/source/resources/cloud_integration.md

```diff
@@ -95,48 +92,55 @@ To run NVTabular on the cloud using GCP, do the following:
    cp -r data/ /mnt/raid/data/
    ```
 
-5. Run the container using the following command:
+5. Run the container by running the following command:
    ```
    docker run --gpus all --rm -it -p 8888:8888 -p 8797:8787 -p 8796:8786 --ipc=host --cap-add SYS_PTRACE -v /mnt/raid:/raid nvcr.io/nvidia/nvtabular:0.3 /bin/bash
    ```
 
 ## Databricks ##
 
-Databricks develops a web-based platform for working with Spark, that provides automated cluster management. Databricks currently support [custom containers](https://docs.databricks.com/clusters/custom-containers.html)
+Databricks has developed a web-based platform on top of Apache Spark to provide automated cluster management. Databricks currently supports 
+[custom containers](https://docs.databricks.com/clusters/custom-containers.html).
 
 To run NVTabular on Databricks, do the following:
 
-1. First we need to create a custom NVTabular container based on Databricks runtime. 
-
-2. Databricks is very sensitive to it dependency which comes natively with the Databricks container runtime. 
```

Comment:
This statement didn't make much sense to me. It also didn't add much value.

@arunraman - Please add some context if you feel like this info is important.